cypress/e2e/12-canvas.cy.ts

```diff
@@ -0,0 +1,286 @@
+		// And last node should be pushed to the right
+		WorkflowPage.getters.canvasNodes().last().should('have.attr', 'style', 'left: 640px; top: 260px;');
+	});
+
+	it('should delete node using node action button', () => {
+		WorkflowPage.actions.addNodeToCanvas(SCHEDULE_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(CODE_NODE_NAME);
+		WorkflowPage.getters
+			.canvasNodeByName(CODE_NODE_NAME)
+			.find('[data-test-id=delete-node-button]')
+			.click({ force: true });
+		WorkflowPage.getters.canvasNodes().should('have.have.length', 1);
+		WorkflowPage.getters.nodeConnections().should('have.length', 0);
+	});
+
+	it('should delete node using keyboard shortcut', () => {
+		WorkflowPage.actions.addNodeToCanvas(SCHEDULE_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(CODE_NODE_NAME);
+		WorkflowPage.getters.canvasNodeByName(CODE_NODE_NAME).click();
+		cy.get('body').type('{backspace}');
+		WorkflowPage.getters.canvasNodes().should('have.have.length', 1);
+		WorkflowPage.getters.nodeConnections().should('have.length', 0);
+	});
+
+	it('should delete node between two connected nodes', () => {
+		WorkflowPage.actions.addNodeToCanvas(SCHEDULE_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(CODE_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(SET_NODE_NAME);
+		WorkflowPage.getters.canvasNodeByName(CODE_NODE_NAME).click();
+		WorkflowPage.actions.zoomToFit();
+		cy.get('body').type('{backspace}');
+		WorkflowPage.getters.canvasNodes().should('have.have.length', 2);
+		WorkflowPage.getters.nodeConnections().should('have.length', 1);
+	});
+
+	it('should delete multiple nodes', () => {
+		WorkflowPage.actions.addNodeToCanvas(SCHEDULE_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(CODE_NODE_NAME);
+		cy.wait(500);
+		WorkflowPage.actions.selectAll();
+		cy.get('body').type('{backspace}');
+		WorkflowPage.getters.canvasNodes().should('have.have.length', 0);
+	});
+
+	it('should move node', () => {
+		WorkflowPage.actions.addNodeToCanvas(MANUAL_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(CODE_NODE_NAME);
+		WorkflowPage.actions.zoomToFit();
+		cy.drag('[data-test-id="canvas-node"].jtk-drag-selected', 50, 150);
+		WorkflowPage.getters.canvasNodes().last().should('have.attr', 'style', 'left: 740px; top: 360px;');
+	});
+
+	it('should zoom in', () => {
+		WorkflowPage.getters.zoomInButton().should('be.visible');
+		WorkflowPage.getters.zoomInButton().click();
+		WorkflowPage.getters.nodeView().should(
+			'have.css',
+			'transform',
+			`matrix(${ ZOOM_IN_X1_FACTOR }, 0, 0, ${ ZOOM_IN_X1_FACTOR }, 0, 0)`
+		);
+		WorkflowPage.getters.zoomInButton().click();
+		WorkflowPage.getters.nodeView().should(
+			'have.css',
+			'transform',
+			`matrix(${ ZOOM_IN_X2_FACTOR }, 0, 0, ${ ZOOM_IN_X2_FACTOR }, 0, 0)`
+		);
+	});
+
+	it('should zoom out', () => {
+		WorkflowPage.getters.zoomOutButton().should('be.visible');
+		WorkflowPage.getters.zoomOutButton().click();
```

Comment:
Whenever you have this pattern, can you combine? `WorkflowPage.getters.zoomOutButton().should('be.visible').click();`

Reply:
👍